userdata/Configuration/GenericWorker/run-hw-generic-worker-and-reboot.bat

```diff
@@ -19,6 +19,7 @@ pushd %~dp0
 set errorlevel=
 C:\generic-worker\generic-worker.exe run --config C:\generic-worker\gen_worker.config >> C:\generic-worker\generic-worker.log 2>&1
 if %errorlevel% equ 67 goto RmLock
+if %errorlevel% equ 1 shutdown /s /t 0 /f /c "Halting as worker is impaired"

```

Comment:
Could the result of the `if` statement on line 21 change the `%errorlevel%` from line 20? In other words, if the `%errorlevel%` is set based on the previous command, and the `if` statement on line 21 qualifies, maybe we should switch lines 21 and 22 to be safe (since line 21 I think is only effective for generic-worker 10 onwards).